flake.nix

```diff
@@ -21,9 +23,6 @@
         packages = { neuron = project.neuron; };
         defaultPackage = packages.neuron;
 
-        apps = { neuron = flake-utils.lib.mkApp { drv = packages.neuron; }; };
-        defaultApp = apps.neuron;
-
```

Comment:
Hmm, why did these two attributes get removed?

Reply:
Oh n/m; just saw your comment on it.

Reply:
Maybe, they should be part of a different PR but the reason for removal is they are unnecessary, since by default `nix run` will try the packages with the same name. App definitions are mainly useful if there are multiple binaries or you want to name them differently.